web/tests/compose.test.cjs

```diff
@@ -5,6 +5,7 @@ const assert = require("node:assert/strict");
 const MockDate = require("mockdate");
 
 const {mock_banners} = require("./lib/compose_banner.cjs");
+const {FakeComposeBox} = require("./lib/compose_helpers.cjs");
```

Comment:
"Fake" is the standard name for a non-real copy of a module for testing, so this is probably OK.